### PR TITLE
Override default NOTIFY on single_script_job to not page on fail

### PR DESCRIPTION
**Pull request by bmedx**

### CODE REVIEW

File: dataeng/jobs/analytics/SqlScripts.groovy [hunk 1]
```diff
@@ -49,6 +49,7 @@ class SqlScripts {
             parameters {
                 stringParam('SOURCE_SCRIPT', '', 'Path to the source script, relative to the root of the repository. Most scripts just require a script name.')
                 stringParam('SCRIPT_NAME')
+                stringParam('NOTIFY', '', 'Space separated list of emails to send notifications to.')
             }
             parameters common_parameters(allVars)
```
Comment (pwnage101) [inline dataeng/jobs/analytics/SqlScripts.groovy]: common_parameters may override the NOTIFY parameter, and even if it doesn't it's double-defined which isn't clean.  My preference would be to either:

1. set the `NOTIFY` variable in `job-configs/SINGLE_SQL_SCRIPT_JOB_EXTRA_VARS.yaml` in the edx/analytics-secure repo to an empty string, or
2. remove the line below `publishers common_publishers(allVars)` which currently has a sole purpose of sending failure notification emails.

Comment (pwnage101) [reply dataeng/jobs/analytics/SqlScripts.groovy]: I don't have a preference either way.  Setting the NOTIFY variable is perhaps more verbose.

Comment (bmedx) [reply dataeng/jobs/analytics/SqlScripts.groovy]: Ah yes I forgot about the job-configs overrides, I can do this there. I don't want to disable notifications in general, people should be able to put in their email if they want to be notified.

**Comment by bmedx** [comment]
Closing in favor of https://github.com/edx-ops/analytics-secure/pull/253

**Pull request closed without merging**